app/routes/[robots.txt].ts

```diff
@@ -0,0 +1,23 @@
+import {LoaderArgs} from '@remix-run/cloudflare'
+
+export const loader = ({request}: LoaderArgs) => {
+  // handle "GET" request
+  // set up our text content that will be returned in the response
+  const origin = new URL(request.url).origin
+  const robotText = `
+User-agent: Googlebot
+Disallow: /nogooglebot/
+
+User-agent: *
+Allow: /
+
+Sitemap: ${origin}/sitemap.xml
```

Comment:
is there a better way to create this file? it's static other than this one url, which really only needs the origin

Reply:
I would say `https://aisafety.info/sitemap.xml` since we don't want to index any other domains 😅

Reply:
what about stampy.ai?

Reply:
Perfect! That makes things a lot simpler